client/apps/shipping-label/view.js

```diff
@@ -239,8 +242,9 @@ class ShippingLabelRootView extends Component {
 	}
 
 	renderLabels() {
-		if ( ! _.every( this.props.shippingLabel.labels, 'statusUpdated' ) ) {
-			return <LoadingSpinner />;
+		if ( this.needToFetchLabelsStatus ) {
```

Comment:
This is a pretty curious change. From what I can see this if statement was introduced ~8 months ago in c21471a2b76dc200d0e947a15356cab9a57c5727 - do you know why it ended up in a merge conflict here?

(No need to solve this before merging, but if you know I'm curious :) )